css/components/header.css

```diff
@@ -0,0 +1,27 @@
+header.container {
+  padding: 0;
+}
```

Comment:
Sakyčiau trūksta, kad headeris butu prisiklijaves prie virsaus.
naudok posision sticky, ir kad visada virsuj (top), ir speju teks uzdeti spalva, naudoti is konstantu base.css

Reply:
Padaryta, panaudojau fixed, nes cia jis labiau tinka